IaC/BareMetal/in-memory-agent.yml

```diff
@@ -0,0 +1,66 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: ngsa
+  labels:
+    app: ngsa
+spec:
+  serviceAccountName: k8s-admin
+  containers:
+  - name: ngsa
+    imagePullPolicy: Always
+    image: retaildevcrew/ngsa:beta
+    args: [ "--in-memory" ]
+    ports:
+    - containerPort: 4120
+
+  - name: logagent
+    imagePullPolicy: Always
+    image: retaildevcrew/logagent:beta
+    env:
+    - name: WorkspaceId
+      valueFrom:
+        secretKeyRef:
+          name: ngsa-secrets
+          key: WorkspaceId
+    - name: SharedKey
+      valueFrom:
+        secretKeyRef:
+          name: ngsa-secrets
+          key: SharedKey
+    - name: LogName
+      value: bartrngsa
```

Comment:
Should **bartrngsa** be referenced from an external value?

Reply:
I'll move it to a config map before I merge.